sdcm/sct_provision/aws/user_data.py

```diff
@@ -39,6 +39,13 @@ def data_device(self) -> DataDeviceType:
             return DataDeviceType.ATTACHED.value
         return DataDeviceType.INSTANCE_STORE.value
 
+    @property
+    def raid_level(self) -> RaidLevelType:
+        """
+        Tell scylla setup to create RAID0 or RAID5 on disks
+        """
+        return self.params.get("raid_level", RaidLevelType.RAID0)
```

Comment:
Wont work, you need to go wth:
```suggestion
        return self.params.get("raid_level") or RaidLevelType.RAID0
```

Reply:
fixed